recipes/libgfortran-abi/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.5.0" %}
+
+package:
+  name: libgfortran-abi
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true                    # [win]
+
+requirements:
+  run:
+    - libgfortran 3.0.0
```

Comment:
You're right. This is not the correct way to guarantee it. How can we properly ensure that we are providing the ABI we claim?